internal/setup/setup.go

```diff
@@ -414,6 +418,16 @@ func parseRelease(baseDir, filePath string, data []byte) (*Release, error) {
 		return nil, fmt.Errorf("%s: no archives defined", fileName)
 	}
 
+	keyringsByName := make(map[string]openpgp.KeyRing, 0)
```

Comment:
I won't repeat this again, but let's please review the PR and update the terminology with some nice var naming (short but neat) so when we're speaking of the content of those fields we refer to them as the archive public keys. If this sounds strange, imagine `age` variable of type `int`. We'd refer to it as the "age" rather than the "int".

Reply:
I've updated the naming where it makes sense. For instance, in `archive_test.go`, where I use the same keyring for creating and verifying signatures, I think it doesn't make sense to call it `PublicKeys`. I don't mind changing it everywhere. Please re-review.

Reply:
I think that makes sense @woky ...if your test key is more than a public key, then it shouldn't be called PublicKey